advocacy_docs/migrating/oracle/migration_journey.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: The database migration “journey”
+---
+
+Migrating your database consists of a nine-step “journey.”
+
+![Migration journey](images/journey.png)
```

Comment:
I think this should be replaced with a version of the migration journey image that only shows the step names and not the bulleted items describing aspects of each step.  If necessary, I can provide a version like the one I'm describing next week after returning from PTO.